clickhouse_backend/models/count.py

```diff
@@ -0,0 +1,93 @@
+from django.db.models import Aggregate
+from django.db.models.expressions import Star
+from django.db.models.fields import IntegerField
+
```

Comment:
1. Maybe renamed to `aggregates.py` following django code structure.
2. Add __all__ variable.